### PR TITLE
doc: remove zero-width space character

### PR DESCRIPTION
**Pull request by mastermay**
This PR removes a zero-width space character in `/zh/docs/contribute/style/hugo-shortcodes/index.md` which caused a rendering error.

/language zh

### CODE REVIEW

**Comment by tengqm** [comment]
Thank you for proposing this fix. However, we will not merge partial fixes as per stated in the [guideline](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87). To find out what are the upstream changes to be synced, do this:

```shell
./scripts/lsync.sh content/zh/docs/contribute/style/hugo-shortcodes/index.md
```

**Comment by mastermay** [comment]
I'll handle it later.

**Comment by mastermay** [comment]
The latest PR contains:
1. synchronize Chinese localization with upstream changes
2. remove Hugo shortcode in comments, which caused a duplicated translation content, see #26748 

@tengqm

**Comment by mastermay** [comment]
btw

A shorcode will get evaluated in the HTML comments, which may cause an error rendering since HTML comments cannot be nested. See example at #26748 

As suggested in the [official document](https://gohugo.io/templates/introduction/#html-comments-containing-go-templates), should we remove all Hugo shortcode in HTML comments?

Is there any way to fundamentally solve the problem once and for all?

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 1]
```diff
@@ -398,6 +348,124 @@ println "This is tab 2."
...
+
+To generate a version string for inclusion in the documentation, you can choose from
+several version shortcodes. Each version shortcode displays a version string derived from
+the value of a version parameter found in the site configuration file, `config.toml`.
+The two most commonly used version parameters are `latest` and `version`.
+-->
+## 版本号信息
+
+文档中的版本号信息可以通过短代码的方式生成。在配置文件 `config.toml` 中，定义了一些版本号信息，通过短代码及版本号参数可以从配置文件中提取并展示指定的版本号信息。
+最常见的版本号短代码参数包括 `latest` 和 `version`。
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:362]: 这个翻译并不是很贴近原文。请试着改进。

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 2]
```diff
@@ -398,6 +348,124 @@ println "This is tab 2."
...
+
+<!--
+### `{{</* param "version" */>}}`
+
+The `{{</* param "version" */>}}` shortcode generates the value of the current version of
+the Kubernetes documentation from the `version` site parameter. The `param` shortcode accepts the name of one site parameter, in this case: `version`.
+-->
+### `{{</* param "version" */>}}`
+
+`{{</* param "version" */>}}` 短代码可以转换为 Kubernetes 文档的当前版本，具体的版本号来源于配置文件中的 `version` 参数。
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:372]:
```suggestion
`{{</* param "version" */>}}` 短代码可以基于站点参数 `version`生成 Kubernetes 文档的
当前版本号取值。
```

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 3]
```diff
@@ -398,6 +348,124 @@ println "This is tab 2."
...
+短代码 `param` 允许传入一个配置参数，例子里的参数为 `version`。
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:373]:
```suggestion
短代码 `param` 允许传入一个站点参数名称，在这里是 `version`。
```

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 4]
```diff
@@ -398,6 +348,124 @@ println "This is tab 2."
...
+
+<!--
+{{< note >}}
+In previously released documentation, `latest` and `version` parameter values are not equivalent.
+After a new version is released, `latest` is incremented and the value of `version` for the documentation set remains unchanged. For example, a previously released version of the documentation displays `version` as
+`v1.19` and `latest` as `v1.20`.
+{{< /note >}}
+-->
+{{< note >}}
+在先前已经发布的文档中，`latest` 和 `version` 参数转换得到的版本号并不相同。
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:383]:
```suggestion
在先前已经发布的文档中，`latest` 和 `version` 参数值并不完全等价。
```

建议贴近原文翻译。这里并不是说版本号不同，相反，版本号是可能相同的，只是这两个参数的含义是不一样的。

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 5]
```diff
@@ -398,6 +348,124 @@ println "This is tab 2."
...
+新版本文档发布后，参数 `latest` 会增加，而 `version` 则保持不变。例如，在上一版本的文档中使用 `version` 会得到 `v1.19`，而使用 `latest` 则会得到 `v1.20`。
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:384]: 请注意较长行的折行。

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 6]
```diff
@@ -398,6 +348,124 @@ println "This is tab 2."
...
+{{< /note >}}
+
+<!--
+Renders to:
+
+{{< param "version" >}}
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:390]: 这两行就没有必要保留在注释中了。反正也没有翻译。

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 7]
```diff
@@ -398,6 +348,124 @@ println "This is tab 2."
...
+<!--
+### `{{</* latest-version */>}}`
+
+The `{{</* latest-version */>}}` shortcode returns the value of the `latest` site parameter.
+The `latest` site parameter is updated when a new version of the documentation is released.
+This parameter does not always match the value of `version` in a documentation set.
+
+Renders to:
+
+{{< latest-version >}}
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:405]: 同上。

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 8]
```diff
@@ -398,6 +348,124 @@ println "This is tab 2."
...
+-->
+### `{{</* latest-version */>}}`
+
+`{{</* latest-version */>}}` 会转换为配置文件中参数 `latest` 对应的值。每当有新版本文档发布时，该参数均会更新。
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:409]:
```suggestion
`{{</* latest-version */>}}` 返回站点参数 `latest` 的取值。每当新版本文档发布时，该参数均会被更新。
```

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 9]
```diff
@@ -398,6 +348,124 @@ println "This is tab 2."
...
+
+The `{{</* latest-semver */>}}` shortcode generates the value of `latest` without the "v" prefix.
+
+Renders to:
+
+{{< latest-semver >}}
+-->
+### `{{</* latest-semver */>}}`
+
+`{{</* latest-semver */>}}` 会转换为配置文件中参数 `latest` 对应的值，并且会删除前缀 `v`。
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:427]: 请斟酌译文。

**Comment by tengqm** [comment]
> A shorcode will get evaluated in the HTML comments, which may cause an error rendering since HTML comments cannot be nested. 

Yes. That is one of the reasons I personally believe that we should use them cautiously. There is currently no generic solution to this yet.

**Comment by mastermay** [comment]
Thanks for your valuable suggestions.

Line breaks in Markdown will be rendered as a space in HTML, which is unnatural in CJK Characters.
I guess we should add line breaks between CJK and Latin characters if possible.

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 10]
```diff
@@ -57,91 +57,42 @@ in Kubernetes version 1.10.
...
+glossary](/docs/reference/glossary/). When the glossary term is moused-over,
+the glossary entry displays a tooltip. The glossary term also displays as a link.
 
 As well as inclusions with tooltips, you can reuse the definitions from the glossary in
 page content.
 -->
 ## 词汇
 
-有两种词汇表提示。
+有两种词汇表提示：<code>glossary_tooltip</code> 和 <code>glossary_definition</code>。
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:91]:
```suggestion
有两种词汇表提示：`glossary_tooltip` 和 `glossary_definition`。
```

**Review by tengqm — COMMENTED** [review]
LGTM other than on nit.

**Pull request merged**